package.json

```diff
@@ -69,8 +64,8 @@
     "jest": "^21.2.1",
     "lint-staged": "^6.1.0",
     "prettier": "1.10.2",
-    "react": "^16.1.1",
-    "react-native": "^0.50.3"
+    "react": "16.8.6",
+    "react-native": "0.60.3"
```

Comment:
Are there any major breaking changes between 60.3, 60.2 and 60.1? If not, I'd prefer this to be `^0.60`.

Reply:
There are no major breaking changes that I know of (in the versions you mentioned). However, there no way to guarantee that. The reason I chose `0.60.3` here is to align it with the example project (we can _update_ it independently in a separate PR). The problem with using `^0.60` here is that RN does not strictly follow semver (yet?). `^` fixes the _major_ version only, so it would happily update it to 0.61.* (or 0.62.x once release etc). And those version _might_ contain breaking changes.